spec/configuration_spec.rb

```diff
@@ -37,7 +37,7 @@
 
         expect(MeiliSearch::Client)
           .to have_received(:new)
-          .with('http://localhost:7700', 's3cr3tap1k3y', {})
+          .with('http://localhost:7700', 's3cr3tap1k3y', { client_agents: 'Meilisearch Rails (v0.7.0)' })
```

Comment:
These three tests will have to be updated at each version change?

Reply:
Oh yeah, good point, I forgot to mock the `qualified_version` :)